pkg/store/filesystem/store.go

```diff
@@ -106,18 +107,30 @@ func (s Store) Write(rec *proto.SettingsBundle) (*proto.SettingsBundle, error) {
 		return nil, fmt.Errorf(emptyKeyError)
 	}
 
-	contents, err := json.Marshal(rec)
-	if err != nil {
-		s.Logger.Err(err).Msg("record could not be marshalled")
+	marshaler := jsonpb.Marshaler{}
+	recordPath := path.Join(s.mountPath, buildFileNameFromBundle(rec))
+	if err := ioutil.WriteFile(recordPath, []byte{}, 0644); err != nil {
```

Comment:
this is a patch. I'm pretty sure there is a way to create a file without needing to write something into it